pkg/kubelet/eviction/eviction_manager_test.go

```diff
@@ -653,7 +653,7 @@ func TestDiskPressureNodeFs(t *testing.T) {
 	podMaker := makePodWithDiskStats
 	summaryStatsMaker := makeDiskStats
 	podsToMake := []podToMake{
-		{name: "low-priority-high-usage", priority: lowPriority, requests: newResourceList("100m", "1Gi", ""), limits: newResourceList("100m", "1Gi", ""), rootFsUsed: "900Mi"},
+		{name: "low-priority-high-usage", priority: lowPriority, requests: newResourceList("100m", "1Gi", ""), limits: newResourceList("100m", "1Gi", ""), logsFsUsed: "900Mi"},
 		{name: "below-requests", priority: defaultPriority, requests: newResourceList("100m", "100Mi", ""), limits: newResourceList("200m", "1Gi", ""), logsFsUsed: "50Mi"},
```

Comment:
@mukesh-mahato Can you give more details about relationship between `logsFsUsed`, `rootFsUsed` and `dedicatedImageFs`?

Reply:
this changes might didn't work, because in https://github.com/kubernetes/kubernetes/blob/8fee46b416c191c9bf86d6ded7d9b087cd48148a/pkg/kubelet/eviction/eviction_manager_test.go#L676
the dedicatedImageFs still set to false